netmap/node_info_test.go

```diff
@@ -22,3 +22,20 @@ func TestNodeInfo_SetAttribute(t *testing.T) {
 	n.SetAttribute(key, val)
 	require.Equal(t, val, n.Attribute(key))
 }
+
+func TestNodeInfo_Status(t *testing.T) {
+	var n NodeInfo
+
+	require.False(t, n.IsOnline())
+	require.False(t, n.IsOffline())
+	require.False(t, n.IsMaintenance())
+
+	n.SetOnline()
+	require.True(t, n.IsOnline())
```

Comment:
Lets also make sure that other states are still `False`? Same below.

Reply:
Fixed